### PR TITLE
Convert `packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBar.test.js` to RTL

### PR DESCRIPTION
**Pull request by xrisaD**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by xrisaD** [comment]
@ashharrison90

File: packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBar.test.js [hunk 1]
```diff
@@ -76,24 +76,26 @@ describe('<SpanBar>', () => {
...
+    expect(getByText(shortLabel)).toBeInTheDocument();
   });
 
   it('log markers count', () => {
     // 3 log entries, two grouped together with the same timestamp
-    const wrapper = mount(<SpanBar {...props} />);
-    expect(wrapper.find(Popover).length).toEqual(2);
+    render(<SpanBar {...props} />);
+    expect(screen.getByTestId('popover')).toBeInTheDocument();
+    expect(screen.getByTestId('popover').querySelectorAll('div')).toHaveLength(2);
```
Comment (ashharrison90) [inline packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBar.test.js:99]:
```suggestion
    render(<SpanBar {...props} />);
    expect(screen.getAllByTestId('SpanBar--logMarker')).toHaveLength(2);
```

**Review by ashharrison90 — CHANGES_REQUESTED** [review]
@xrisaD thanks again for this 😄 

couple of suggestions. you'll also need to update the selector here so that the e2e tests continue to pass: https://github.com/grafana/grafana/blob/main/packages/grafana-e2e-selectors/src/selectors/components.ts#L268 to be `spanBar: () => 'data-testid SpanBar--wrapper'`

in terms of the enterprise check, it's a fairly new thing we've added. i'm not sure if it works for contributions from forks 🤔 don't worry about it - it's something i'll need to look into. 👍

**Comment by xrisaD** [comment]
@ashharrison90 thank you for the code review! 😄

**Review by ashharrison90 — APPROVED** [review]
looks great! 🥳

**Comment by ashharrison90** [comment]
@xrisaD just looking into why the e2e tests are failing... probably something silly i forgot 😅

**Comment by ashharrison90** [comment]
@xrisaD i pushed the fix directly in https://github.com/grafana/grafana/pull/49107/commits/afc80e2fbfe4bb599f0b98f54b6f7d8555ec9c6e as i was struggling to think of a nice way of explaining it. hope you don't mind 😅 

feel free to check it out 😄

**Pull request merged**